kratos/python/add_utilities_to_python.cpp

```diff
@@ -74,6 +74,77 @@ void SetOnProcessInfo(
     rCurrentProcessInfo[TABLE_UTILITY] = pTable;
 }
 
+double RotateAndCallFunctionCurrentConfiguration(
+    PythonGenericFunctionUtility& rPythonGenericFunctionUtility,
+    const double x,
+    const double y,
+    const double z,
```

Comment:
Wouldn't it be better to use a Point or array_1d<double,3> instead of coordinates separately?

Reply:
Yes..., but the initial implementation was with X, Y, Z and to avoid break things I left it that way

Reply:
I see....